### ISSUE TITLE
"Remove text for hearing impaired" seems not to be working in V.3.4.13 (64-Bit)

### ISSUE THREAD

**Issue by Whisker72** [SubtitleEdit/subtitleedit#1851]
1) The issue:
When I want to remove text for hearing impaired (parentheses, text before colon etc.) everything seems to work as it used to when I click OK or Apply (i.e. the list of lines in the dialog which would be removed is empty).
But after the dialog is closed, nothing at all is actually removed, every line is still there.
I tried it multiple times on the same .SRT file and with various .SRT files and it's always the same behaviour.

2) BTW:
It would be nice to have a button in the toolbar for this function, as I'm using this feature quite a lot. Actually, it would be really nice to have a more configurable toolbar. I.e. being able to put more menu functions in the toolbar and maybe also being able to change the order of the buttons as well as having fixed and variable spacers as well as dividers.
So please consider 2) a feature request for some future version (not critical, but "nice to have"). :wink:

3) And another thing: Kudos again for the good work. Just out of curiosity, I tried a few other subtitle editors lately, and still none even comes close to yours. :+1:

**Comment by ivandrofly** [comment]
> But after the dialog is closed, nothing at all is actually removed, every line is still there.

Can you provide some examples?

**Comment by niksedk** [comment]
@Whisker72: 
1) Sorry about the bug in the "OK" / "Apply"... it should be fixed in latest beta: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.4.13/SubtitleEditBeta.zip
Please do test it :)

2) He, the hardest part is finding an icon... 

3) thx - we do try :)

**Comment by Whisker72** [comment]
1) Will do. For the time being, I went back to 3.4.12 where the problem doesn't occur, but I'll test it and get back to you.

2) I just hacked together a quick PNG based on the symbol for inductive audio loops. Feel free to use it if you like, I reserve no rights (not sure if I could anyway because I used the sign for audio induction loops as a basis).

_Hmmm, upload doesn't work, would it be okay if I throw it over via mail?_
